static/js/components/email/AutomaticEmailOptions_test.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import React from 'react';
+import sinon from 'sinon';
+import { mount } from 'enzyme';
+import { assert } from 'chai';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+
+import AutomaticEmailOptions from './AutomaticEmailOptions';
+
+describe('AutomaticEmailOptions', () => {
+  let sandbox, setEmailCompositionTypeStub;
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+    setEmailCompositionTypeStub = sandbox.stub();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  const renderComponent = (sendAutomaticEmails: boolean = false) => (
+    mount(
+      <MuiThemeProvider muiTheme={getMuiTheme()}>
+        <AutomaticEmailOptions
+          setSendAutomaticEmails={setEmailCompositionTypeStub}
+          sendAutomaticEmails={sendAutomaticEmails}
+        />
+      </MuiThemeProvider>
+    )
+  );
+
+  it('div renders', () => {
+    let wrapper = renderComponent();
+    assert.equal(wrapper.find(".type-radio-group").children().length, 2);
+  });
+
+  it('div renders for type: one time email', () => {
```

Comment:
could you rename this test to just be about firing the `onChange` handler? also generally we try to name tests something like `it('should render a div')`.